setup.cfg

```
@@ -32,7 +32,7 @@ follow_imports = skip
 check_untyped_defs = True
 warn_unused_ignores = True
 strict_optional = False
-no_implicit_optional = True
+warn_redundant_casts = True
```

Please discuss the config change in a different PR/issue.
```suggestion
no_implicit_optional = True
warn_redundant_casts = True
```